moecache.py

```diff
@@ -353,25 +371,32 @@ def get(self, key):
         self._validate_key(key)
 
         command = 'get ' + key + '\r\n'
+        command = bytearray(command, 'utf-8')
         val = None
         node = self._find_node(key)
         resp = node.send(command)
         error = None
 
         # make sure well-formed responses are all consumed
-        while resp != 'END\r\n':
+        while resp != bytearray('END\r\n', 'utf-8'):
             terms = resp.split()
-            if len(terms) == 4 and terms[0] == 'VALUE':  # exists
+            if (
+                len(terms) == 4 and
+                terms[0] == bytearray('VALUE', 'utf-8')
+            ):  # exists
                 typecode = int(terms[2]) ^ 0x100
                 length = int(terms[3])
                 if typecode > 0xff:
                     error = ClientException('not a moecache deployment')
-                if terms[1] == key:
+                if terms[1] == bytearray(key, 'utf-8'):
                     received = node.gets(length+2)[:-2]
                     if typecode == 18:
-                        val = received
+                        val = received.decode('utf-8')
```

Comment:
`val`'s type is changed to `unicode` in Python2, thus a potential break.  I suggest to leave it as is.